WcaOnRails/config/locales/ru.yml

```diff
@@ -1402,11 +1403,11 @@ ru:
     nearby_competitions:
       #original_hash: 241801a
       competitions:
-        other: '%{count} WTF'
-        many: '%{count} соревнований'
-        few: '%{count} соревнования'
-        one: '%{count} соревнование'
         zero: Нет соревнований
+        one: '%{count} соревнование'
+        few: '%{count} соревнования'
+        many: '%{count} соревнований'
+        other: '%{count} WTF'
```

Comment:
It's for decimals probably, see https://github.com/svenfuchs/rails-i18n/issues/703#issuecomment-273335796. In Polish I just copy other as it doesn't matter.